contracts/nft/src/execute/revoke_all.rs

```diff
@@ -0,0 +1,16 @@
+use cosmwasm_std::{Addr, DepsMut, Env, MessageInfo, Response};
+
+use crate::core::error::ContractError;
+use crate::storage::operators::OPERATORS;
+use crate::util::action::{Action, ActionType};
+
+pub fn handle(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    operator: Addr,
+) -> Result<Response, ContractError> {
+    OPERATORS.remove(deps.storage, (&info.sender, &operator));
```

Comment:
Same as my comment from approve_all. Do we need any sort of permission check?

Reply:
not for this one since it's the sender is removing permission